samples/display/ili9340/src/main.c

```diff
@@ -49,15 +50,15 @@ void main(void)
 	dev = device_get_binding("ILI9340");
 
 	if (dev == NULL) {
-		SYS_LOG_ERR("device not found.  Aborting test.");
+		LOG_ERR("device not found.  Aborting test.");
```

Comment:
Could you fix log message:
- device->Device 
- one redundant space before 'Aborting'.